apps/etterlatte-statistikk/src/main/resources/db/migration/V52__rydd_saksstatistikk.sql

```diff
@@ -0,0 +1,37 @@
+-- Fiks resultat tilbakekreving avbrutt
+update sak
+set behandling_resultat = 'AVBRUTT'
+where behandling_status = 'AVBRUTT'
+  and behandling_resultat is null
+  and behandling_type = 'TILBAKEKREVING';
+
+-- Fiks registrert tidspunkt klage
+update sak s
+set registrert_tid = foerste_paa_klage.rtid
+from (select min(registrert_tid) as rtid, behandling_id
+      from sak
+      where behandling_type = 'KLAGE'
+      group by behandling_id) as foerste_paa_klage
+where s.behandling_id = foerste_paa_klage.behandling_id;
+
+-- Fiks status klager vi har sendt til kabal
+-- Ønske om en annen mapping for de som vi sender til kabal
+update sak
+set behandling_status = 'OVERSENDT_KA'
+where behandling_type = 'KLAGE'
+  and behandling_status = 'FERDIGSTILT'
+  and behandling_resultat = 'STADFESTE_VEDTAK';
+
+-- fix forsvinnende utlandstilknytning revurdering
+-- manglende oppdatering i påfølgende meldinger historisk, feilen er fikset men denne patcher
+-- gamle saker som ligger i databasen
+update sak s
+set sak_utland = fasit.sak_utland
```

Comment:
Hvordan blir dette nå som utlandstilknytning egentlig ligger på behandling? Vil nå kunne endre seg i en revurdering

Reply:
Det vil kunne endre seg, men dette er for å rydde opp i meldinger der dette ikke ble sendt med riktig tilbake i tid (jeg tror heller ikke du "får lov" til å fjerne en utlandstilknytning, bare endre den -- men disse er registrert med `null`)